src/Agent.Plugins/Agent.Plugins.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Agent.Sdk\Agent.Sdk.csproj" />
+    <ProjectReference Include="..\Microsoft.VisualStudio.Services.Agent\Microsoft.VisualStudio.Services.Agent.csproj" />
```

Comment:
plugins should not take a dependency on `Microsoft.VisualStudio.Services.Agent\Microsoft.VisualStudio.Services.Agent.csproj` since the `Microsoft.VisualStudio.Services.Agent.csproj` also has a dependency on `SdK.csproj`, what method you used from it? can you port it into the SDK?

Reply:
followed up on Teams - I'm using IHostContext to get ProcessInvoker hostContext.CreateService<IProcessInvoker>() and use  processInvoker.ExecuteAsync to run robocopy command